### ISSUE TITLE
[Adaptive Card - InputsWithValidation.json]: Auto scroll is not working while navigate on the input with validation screen. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7594]
**Test Environment:**
Device: iPhone 11
iOS: 15.5
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.20220612.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.3" control and select it.
5. Navigate to 'Scenarios' control and activate it.
6. Navigate to 'InputWithValidation.json' and activate it.
7. Perform right swipe and observe the issue.

Actual Result:
Auto scroll is not working when navigating on the input with validation screen using swipe gesture.

**Expected Result:**
Auto scroll should be work properly when navigate the screen through the swipe gesture.

**User Experience:**
If the auto scroll will not work on the Inputs with validation screen then it will impact the screen reader user as they will get confused about the elements.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://user-images.githubusercontent.com/95913748/176499387-7b842aa7-8dc2-4bfb-aed7-39bb5f9b5d30.MP4

**Comment by jwoo-msft** [comment]
Hi @vagpt,
The auto scrolling function has to be provided by sample app.
AdaptiveCards does not have scrolling function.
Since sample is not facing outside audience. I will close this issue.

**Comment by vagpt** [comment]
Closing this issue as this is not in our scope.
